sge/chain.json

```diff
@@ -182,9 +182,9 @@
   },
   "explorers": [
     {
-      "kind": "nodeist.net",
-      "url": "https://exp.nodeist.net/Sge",
-      "tx_page": "https://exp.nodeist.net/Sge/tx/${txHash}"
+      "kind": "sge",
```

Comment:
I think 'kind' should be blockexplorer, or ping pub?

Reply:
@JeremyParish69 Please review